theme/banner/_base.scss

```diff
@@ -37,7 +38,7 @@
 	}
 
 	a {
-		color: #fff;
+		color: $wb-lng-glb-mn--pnl-btn-a-color;
```

Comment:
Reference to ``$wb-lng-glb-mn--pnl-btn-a-color`` should be renamed to ``$wb-lng-glb-mn-pnl-btn-a-color``.